### ISSUE TITLE
BrokenPipeError with clisp2.49 python3.5

### ISSUE THREAD

**Issue by pulinagrawal** [marcoheisig/cl4py#5]
```
>>> import cl4py
>>> lisp=cl4py.Lisp(cmd=['clisp'])
>>> lisp.eval( ('+', 2, 3) )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/cl4py/lisp.py", line 50, in eval
    self.stdin.write(sexp + '\n')
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by marcoheisig** [comment]
Oh, sorry about that.

Admittedly, I have never tested cl4py with Clisp.  Let me investigate this...

... ok, just some minor issues.  Fixed in 991e2cbfd6c081423b20cb710386877174384b66.

I will upload the improved version to PyPi when I return from vacation - sometime next week.

**Comment by pulinagrawal** [comment]
Wow! that was quick. Thanks a lot. Can you put a howler out when you upload the update to PyPi?
